build/source/engine/HDS.f90

```diff
@@ -0,0 +1,321 @@
+module HDS
+    USE nrtype
+    ! physical constants
+    USE multiconst,only:LH_vap,  & ! latent heat of vaporization   (J kg-1)
+                        iden_water ! intrinsic density of water    (kg m-3)
+    
+    contains
+    subroutine init_summa_HDS(pondVolFrac, depressionDepth, depressionAreaFrac, totalArea, p, pondVol, pondArea, conArea, vMin)
+        ! initialize HDS pothole storage model states for SUMMA
+        implicit none
+        !subroutine arguments
```

Comment:
Have as local variables instead of subroutine arguments, for example:
`associate(pondVolFrac => var(iLookBVAR%pondArea)%dat(1))`

Reply:
Done